Automattic/wp-calypso
# Staging Sites: Hide module on current staging sites

Fixes https://github.com/Automattic/dotcom-forge/issues/1921

## Proposed Changes

Hides the 'Staging site' module on 'Hosting Configuration' if the site is a staging site.

We made eventually add some UI for this with https://github.com/Automattic/dotcom-forge/issues/1929 but better to hide for now.

## Testing Instructions

1. Navigate to Hosting Configuration for a production site.
2. Verify the 'Staging site' module is still present.
3. Navigate to Hosting Configuration for a staging site.
4. Verify 'Staging site' module is absent.
